bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tmva/__init__.py

```diff
@@ -45,7 +43,7 @@ def inject_rbatchgenerator(ns):
 
 from ._gnn import RModel_GNN, RModel_GraphIndependent
 
-hasRDF = "dataframe" in gROOT.GetConfigFeatures()
+hasRDF = "dataframe" in cppyy.gbl.ROOT.GetROOT().GetConfigFeatures()
```

Comment:
Is this change necessary?  If it is, does it also imply that user code currently using `gROOT` would need to change?

Reply:
Yes, it's necessary because we don't import `gROOT` from `libROOTPythonizations` anymore (since I removed the reference in there). However, users don't have to change anything, because `gROOT` is still set in the ROOT facade. However, this code here is not using the ROOT modulde/facade. It's an implementation detail of the pythonizations, which also used the internal `libROOTPythonizations` before. Hence the change is required here but not in usercode, which does `ROOT.gROOT`.

Reply:
I see
> from cppyy.gbl import gSystem
then I suspect
```
from cppyy.gbl import gROOT
```
would also work, right?
or this would also work, right?:
```
in cppyy.gbl.gROOT.GetConfigFeatures()
```
(not asking to change the code, just checking I understand correctly)